src/core/DataTable/exportCSV.js

```diff
@@ -0,0 +1,37 @@
+function convertToCSV(objArray) {
+  return (typeof objArray !== 'object' ? JSON.parse(objArray) : objArray)
+    .map(innerArray => innerArray.filter(val => val).join(','))
+    .join('\r\n');
+}
+
+export default function exportCSV(headers, items, fileTitle) {
+  if (headers) {
+    items.unshift(headers);
```

Comment:
This modifies the original data... Clicking it multiple times leads to repeated header
![Screenshot from 2020-02-04 16-56-23](https://user-images.githubusercontent.com/1779590/73751402-1a37df80-4770-11ea-86fb-7ee9ac106425.png)